metrics/bigquery.py

```diff
@@ -28,10 +26,10 @@
 import time
 import traceback
 
-import influxdb
 import requests
 import ruamel.yaml as yaml
 
+BACKFILL_DAYS = 30
```

Comment:
I personally think I'll want this to be a flag again at some point but this is good for now